cmd/mt-gateway/main.go

```diff
@@ -23,6 +26,14 @@ var (
 	importerURL   = flag.String("importer-url", "", "mt-whisper-importer-writer address")
 	defaultOrgId  = flag.Int("default-org-id", -1, "default org ID to send to downstream services if none is provided")
 	brokers       = flag.String("kafka-tcp-addr", "localhost:9092", "kafka tcp address(es) for metrics, in csv host[:port] format")
+
+	// stats
+	statsEnabled    = flag.Bool("stats-enabled", false, "enable sending graphite messages for instrumentation")
+	statsPrefix     = flag.String("stats-prefix", "tsdb-gw.stats.default.$hostname", "stats prefix (will add trailing dot automatically if needed)")
```

Comment:
the default value here does not match the one in the config file. I think it should also be `mt-gateway.stats.default.$hostname`

Reply:
oops, e5ee174d386d81815f60907794fea32d068562ef